mkosi/distributions/centos.py

```diff
@@ -98,16 +98,36 @@ def architecture(cls, arch: Architecture) -> str:
 
     @staticmethod
     def gpgurls(context: Context) -> tuple[str, ...]:
+        # First, start with the names of the appropriate keys in /etc/pki/rpm-gpg.
+
+        if context.config.release == "9":
+            rel = "RPM-GPG-KEY-centosofficial"
+        else:
+            rel = "RPM-GPG-KEY-centosofficial-SHA256"
+
+        one = find_rpm_gpgkey(context, rel, required=False)
+
+        # Next, follow up with the names of the appropriate keys in /usr/share/distribution-gpg-keys.
+
         if context.config.release == "9":
             rel = "RPM-GPG-KEY-CentOS-Official"
         else:
             rel = "RPM-GPG-KEY-CentOS-Official-SHA256"
 
-        return tuple(
-            find_rpm_gpgkey(context, key, f"https://www.centos.org/keys/{key}")
-            for key in (rel, "RPM-GPG-KEY-CentOS-SIG-Extras")
+        # The key in /usr/share/distribution-gpg-keys is only required if we didn't find one in
+        # /etc/pki/rpm-gpg.
+        two = find_rpm_gpgkey(context, rel, f"https://www.centos.org/keys/{rel}", required=one is None)
```

Comment:
Easier to parse visually
```suggestion
        two = find_rpm_gpgkey(context, rel, f"https://www.centos.org/keys/{rel}", required=(one is None))
```